tests/testthat/test_summarize_copilot.R

```diff
@@ -469,11 +470,11 @@ describe('map_responses_to_cycles', {
   )
 
   triton.cycle <- tribble(
-    ~uid,       ~team_id,     ~start_date,  ~extended_end_date,    ~ordinal,
-    'Cycle_1',  'Team_Viper', '2020-01-01', '2020-01-14',          1,
-    'Cycle_2',  'Team_Viper', '2020-01-15', '2020-01-30',          2,
-    'Cycle_3',  'Team_Fox',   '2020-01-01', '2020-01-14',          1,
-    'Cycle_4',  'Team_Fox',   '2020-01-15', '2020-01-30',          2
+    ~uid,       ~team_id,     ~start_date,  ~extended_end_date,    ~ordinal,  ~extended_start_date,
+    'Cycle_1',  'Team_Viper', '2020-01-01', '2020-01-14',          1,         '',
+    'Cycle_2',  'Team_Viper', '2020-01-15', '2020-01-30',          2,         '',
+    'Cycle_3',  'Team_Fox',   '2020-01-01', '2020-01-14',          1,         '',
+    'Cycle_4',  'Team_Fox',   '2020-01-15', '2020-01-30',          2,         '',
```

Comment:
nit: this example data is surprising because Copilot is set to _always_ define `extended_start_date` for (and only for) ordinal-1 cycles. In fact that behavior is already happening in production, so we don't have to worry about old-style dates where `extended_start_date` is missing. It's fine if you're proving the RServe is actually more robust than that, but it might give readers the wrong impression.

Reply:
9707589